### PR TITLE
NI : Add some channels.

### PR DESCRIPTION
**Pull request by BellezaEmporium**

### CODE REVIEW

File: channels/ni.m3u [hunk 1]
```diff
@@ -5,3 +5,7 @@ https://cootv.cootel.com.ni:443/streams/d/SSLCH10-SD-ZUK/playlist.m3u8
 https://cootv.cootel.com.ni:443/streams/d/SSLCH11-SD-GZN/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTawbZLCedJzRX06CAyU-jX37n9iyTxMVcIfg&usqp=CAU" group-title="",CANAL 9
 https://cootv.cootel.com.ni:443/streams/d/SSLCH09-HD-CVS/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",VIVA NICARAGUA CANAL 13
+https://59802217c6db4.streamlock.net/ch13/videoch13/chunklist_w1505653761_DVR.m3u8
```
Comment (freearhey) [inline channels/ni.m3u:9]: Geo-blocked

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

File: channels/ni.m3u (hunk 1 above)
Comment (Dum4G) [reply channels/ni.m3u:9]: image
> Geo-blocked

I was able to get a single chunk without any VPN but something is wrong with https sertificate

Comment (LaneSh4d0w) [reply channels/ni.m3u:9]: Yeah there's a certificate issue. I'm going to look at it.

**Review by freearhey — APPROVED** [review]

**Comment by Dum4G** [comment]
Still returns me a single 8 seconds chunk that was shown a long time ago

**Comment by LaneSh4d0w** [comment]
> 
> 
> Still returns me a single 8 seconds chunk that was shown a long time ago

Let me see.

**Pull request merged**